go/cmd/dolt/commands/dump.go

```diff
@@ -78,6 +84,8 @@ func (cmd DumpCmd) createArgParser() *argparser.ArgParser {
 	ap := argparser.NewArgParser()
 	ap.SupportsFlag(forceParam, "f", "If data already exists in the destination, the force flag will allow the target to be overwritten.")
 	ap.SupportsString(FormatFlag, "r", "result_file_type", "Define the type of the output file. Valid values are sql and csv. Defaults to sql.")
+	ap.SupportsString(filenameFlag, "", "file_name", "Define file name for dump file.")
```

Comment:
Mention defaults for these two

Reply:
And make sure you use constants rather than hard-coding the current string values

Reply:
got it!